libraries/botframework-connector/src/auth/authenticationConstants.ts

```diff
@@ -83,14 +83,34 @@ export namespace AuthenticationConstants {
     export const ToBotFromEmulatorOpenIdMetadataUrl =
         'https://login.microsoftonline.com/common/v2.0/.well-known/openid-configuration';
 
+    /**
+     * The V1 Azure AD token issuer URL template that will contain the tenant id where the token was issued from.
+     */
+    export const ValidTokenIssuerUrlTemplateV1 = 'https://sts.windows.net/';
+
+    /**
+     * The V2 Azure AD token issuer URL template that will contain the tenant id where the token was issued from.
+     */
+    export const ValidTokenIssuerUrlTemplateV2 = 'https://login.microsoftonline.com/';
+
+    /**
+     * The Government V1 Azure AD token issuer URL template that will contain the tenant id where the token was issued from.
+     */
+    export const ValidGovernmentTokenIssuerUrlTemplateV1 = 'https://login.microsoftonline.us/';
+
+    /**
+     * The Government V2 Azure AD token issuer URL template that will contain the tenant id where the token was issued from.
+     */
+    export const ValidGovernmentTokenIssuerUrlTemplateV2 = 'https://login.microsoftonline.us/';
```

Comment:
Shouldn't this be in the [`GovernmentConstants`](https://github.com/microsoft/botbuilder-js/blob/eb58d15b67405fe11a9bcdf958ff5a241feced97/libraries/botframework-connector/src/auth/governmentConstants.ts) instead? @msomanathan 

It's not a big deal, but historically all of the US Gov Cloud related constants are stored in a separate file. My main concern is the v4-to-v5 work and making sure our ducks are in a row.

I did note that this is a verbatim port from [.NET](https://github.com/microsoft/botbuilder-dotnet/pull/5829/files#diff-7f25611536bab084126e2ba4ea408532cc78ac0b18534e36365ab3c7c7cd71b1R88-R96).

Reply:
Filed https://github.com/microsoft/botframework-sdk/issues/6384 for tracking.